test/browser/features/fixtures/network_breadcrumbs/script/fetch.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <script src="/node_modules/@bugsnag/browser/dist/bugsnag.min.js"></script>
+    <script type="text/javascript">
+      var NOTIFY = decodeURIComponent(window.location.search.match(/NOTIFY=([^&]+)/)[1])
+      var SESSIONS = decodeURIComponent(window.location.search.match(/SESSIONS=([^&]+)/)[1])
+      var API_KEY = decodeURIComponent(window.location.search.match(/API_KEY=([^&]+)/)[1])
+
+      Bugsnag.start({
+        apiKey: API_KEY,
+        endpoints: { notify: NOTIFY, sessions: SESSIONS },
+      })
+
+      Promise.all([
+        fetch("fetch.html"),
+        fetch("/i_dont_exist.html")
+      ]).then(() => {
+        throw new Error("I should have network breadcrumbs attached")
+      })
+    
+    </script>
+  </head>
+  <body>
+    <h1>Fetch tests</h1>
+    <pre id="bugsnag-test-should-run">PENDING</pre>
+    <script>
+      var el = document.getElementById('bugsnag-test-should-run')
+      el.textContent = el.innerText = (typeof Promise !== 'undefined' && typeof Promise.all !== 'undefined' && 'fetch' in window)
```

Comment:
what is `el.textContent = el.innerText` doing out of interest? (I haven't reviewed many maze runner test PRs)

Reply:
I think we're assigning Yes/No to both `textContent` and `innerText` to support multiple types of html tags, I'm confident that `textContent` would cover all situations though, I was simply following suit!